container-host-files/etc/scf/config/role-manifest.yml

```diff
@@ -32,7 +32,7 @@ instance_groups:
               tag: mysql-data
               size: 20
           shared-volumes: []
-          memory: 1900
+          memory: 1400
```

Comment:
The data has a max of 456; I guess we're still tripling this? (The nearest round number is 1536)

Reply:
Yes, with some rounding up